### PR TITLE
[Ember-Migration] Created "Back to Home" button

### LINKED ISSUE #401: [Ember-Migration] Create a back to home button
## Issue
- When user arrives on last step of Join Section, they have nothing to navigate to home, other than navigation bar
image

## Suggested Solution  
- Create a button with text `Back to Home` below the next steps
- Use the existing reusable components for the button
- Use `transitionTo` to navigate to home page

## For contributing
- Make sure that the PR is raised to `develop-ember` not `develop` as base branch
- https://github.com/Real-Dev-Squad/website-www/blob/develop-ember/CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by sahsisunny**
## Issue
Closes #401

### What is the change?

- `app/components/join-steps/step-five.hbs` - Using reusable components, create a button with text `Back to Home`
- `app/components/join-steps/step-five.js` - Use `this.router.transitionTo()` to redirect to the home page
- `app/styles/stepper.module.css` - Add styles for the button container
- `app/router.js` - Add a index route for the home page
- `app/routes/index.js` - Add a route for the home page
- `tests/unit/routes/index-test.js` - Add a test for the home page route
- `tests/integration/components/step-five-test.js` - Add a test for the button

### Is Development Tested?

- [X] Yes
- [ ] No

### After Change Screenshots

Screenshot 2023-02-05 191102

### CODE REVIEW

File: app/components/join-steps/step-five.js [hunk 1]
```diff
@@ -22,4 +22,8 @@ export default class StepFiveComponent extends Component {
   @action onError() {
     this.toast.error('Error in copying to clipboard', 'Error!', TOAST_OPTIONS);
   }
+
+  @action backToHome() {
+    window.location.href = APPS.HOME;
```
Comment (rohan09-raj) [inline app/components/join-steps/step-five.js:27]: This won't work here as it's an SSR (server-side rendered) application, therefore we won't be able to access document and window properties here as the website is loaded from the server. You can explicitly handle the null and undefined before using it. 
I would recommend using the ember router service here and use it's method `this.router.transitionTo`

Comment (rohan09-raj) [reply app/components/join-steps/step-five.js:27]: https://guides.emberjs.com/release/routing/redirection/

**Review by rohan09-raj — CHANGES_REQUESTED** [review]

File: tests/integration/components/step-five-test.js [hunk 2]
```diff
@@ -36,5 +36,9 @@ module('Integration | Component | step-five', function (hooks) {
       .hasText(
         'If the link is verified you will hear back with a personalized joining link within 7 working days'
       );
+    assert
+      .dom('[data-test-button="back-to-home"]')
+      .hasText('Back to Home')
+      .exists();
```
Comment (rohan09-raj) [inline tests/integration/components/step-five-test.js:42]: Write an assert for redirection also

Comment (sahsisunny) [reply tests/integration/components/step-five-test.js:43]: added for route

File: app/components/join-steps/step-five.js (hunk 1 above)
Comment (sahsisunny) [reply app/components/join-steps/step-five.js]: done

**Review by satyam73 — CHANGES_REQUESTED** [review]

File: app/components/join-steps/step-five.hbs [hunk 3]
```diff
@@ -30,4 +30,13 @@
       </li>
     </ol>
   </div>
+  <div class='button__container'>
```
[the remark this answers is not in the document]
Comment (rohan09-raj) [reply app/components/join-steps/step-five.hbs:33]: @satyam73 Won't this effect the alignment of rest of the elements which are left aligned?

File: tests/integration/components/step-five-test.js (hunk 2 above)
Comment (rohan09-raj) [reply tests/integration/components/step-five-test.js:43]: @sahsisunny We can test the button as follows :
- Check that current route is `/join`
- Click the button
- Check that current route is `/`

File: app/components/join-steps/step-five.hbs (hunk 3 above)
[the remark this answers is not in the document]
Comment (satyam73) [reply app/components/join-steps/step-five.hbs:33]: Which element you're talking about?

[the remark this answers is not in the document]
Comment (rohan09-raj) [reply app/components/join-steps/step-five.hbs:33]: The inner text elements here which are left aligned won't they become center aligned ??
I think, I got my answer, they won't be affected as the elements are in a `div` and the property will be applied to `div` not the elements in that `div`
image

[the remark this answers is not in the document]
Comment (satyam73) [reply app/components/join-steps/step-five.hbs:33]: > The inner text elements here which are left aligned won't they become center aligned ?? I think, I got my answer, they won't be affected as the elements are in a `div` and the property will be applied to `div` not the elements in that `div`
> 
> Can you verify the same?
> 
> image

Yep you're right
Take a look into this
image

[the remark this answers is not in the document]
Comment (rohan09-raj) [reply app/components/join-steps/step-five.hbs:33]: Okay great @sahsisunny, we can remove the container as it's not required as well as the CSS linked to that and add the required CSS in `.joining` class

File: app/styles/stepper.module.css [hunk 4]
```diff
@@ -82,6 +82,11 @@
   padding-left: 20px;
 }
 
+.button__container {
```
[the remark this answers is not in the document]
Comment (sahsisunny) [reply app/styles/stepper.module.css]: removed

File: app/components/join-steps/step-five.hbs (hunk 3 above)
[the remark this answers is not in the document]
Comment (sahsisunny) [reply app/components/join-steps/step-five.hbs]: add css properties to `joining` class and removed container.

File: tests/integration/components/step-five-test.js (hunk 2 above)
Comment (sahsisunny) [reply tests/integration/components/step-five-test.js:43]: added using acceptance

**Review by rohan09-raj — APPROVED** [review]

**Review by satyam73 — APPROVED** [review]
LGTM👍
Good job🎉

**Pull request merged**